build.yaml

```diff
@@ -1842,47 +1852,57 @@ steps:
   - kind: runImage
     name: delete_batch_tables
     image:
-      valueFrom: admin_pod_image.image
+      valueFrom: batch_image.image
     script: |
       set -ex
-      mysql --defaults-extra-file=/sql-config/sql-config.cnf < /io/sql/delete-batch-tables.sql
+      cd /io
+      python3 delete_dev_database.py {{ default_ns.name }} batch
     inputs:
-      - from: /repo/batch/sql
-        to: /io/sql
+      - from: /repo/ci/delete_dev_database.py
+        to: /io/delete_dev_database.py
     secrets:
       - name: database-server-config
         namespace:
           valueFrom: default_ns.name
         mountPath: /sql-config
+    serviceAccount:
+      name: admin
+      namespace:
+        valueFrom: default_ns.name
     runIfRequested: true
     scopes:
       - dev
     dependsOn:
       - default_ns
-      - admin_pod_image
+      - batch_image
```

Comment:
Why does this depend on the batch_image? And ci_image below and monitoring_image above?

Reply:
Ah right, I forgot to mention. `admin-pod` doesn't have `gear` installed, only the mysql CLI client. I could have used `ci-utils-image` but I figured you're most likely to run these steps like `delete_batch_tables,deploy_batch` so better to use an image that is probably already in the pipeline.